ddl/schema_test.go

```diff
@@ -146,13 +146,13 @@ func (s *testSchemaSuite) TestSchema(c *C) {
 		_, err := tbl1.AddRecord(ctx, types.MakeDatums(i, i, i), false)
 		c.Assert(err, IsNil)
 	}
-	// create table t1 with defaultBatchCnt+10 records.
+	// create table t1 with 1034 records.
```

Comment:
Please add comments for 1034, I though it was a typo of 1024 at first impression.

Reply:
`defaultBatchCnt` doesn't be used here. So 1034 has no meaning. I do not know what to write.